pages/programs/index.tsx

```diff
@@ -1,23 +1,123 @@
-import { ProgramsList } from '../../lib/components/programs-list';
-import { Text, Container, Flex } from '@chakra-ui/react';
+import { ProgramListingProps, ProgramsList } from '../../lib/components/programs-list';
+import { Container, Flex } from '@chakra-ui/react';
 import OSLHeader from '../../lib/components/header';
 
+const programs: ProgramListingProps[] = [
+  {
+    title: 'MONTHLY WRITING GROUP',
+    description:
```

Comment:
Is this really the best way to store this information? At this point a line by line string is probably a bad representation.